### PR TITLE
Prepare test release 1.47.1

### PR DESCRIPTION
**Pull request by nicodh**

### CODE REVIEW

**Review by r10s — APPROVED** [review]

**Review by Simon-Laux — CHANGES_REQUESTED** [review]
you forgot to run `pnpm -w update:target-versions`

**Comment by nicodh** [comment]
Ah ok, I already wondered why pnpm install didn't update anything...

**Review by Simon-Laux — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,8 +2,19 @@
...
+
+## Fixed
+
+<a id="1_47_1"></a>
+
+## [1.47.1] - 2024-11-01
+
 ## Added
 - Added support for selecting multiple files in the attachment file picker. #4278
+- Several accessibility improvements for keyboard navigation
```
Comment (WofWca) [inline CHANGELOG.md:17]: There is already a line about this below `improve keyboard and screen-reader accessibility #4210`

Comment (nicodh) [reply CHANGELOG.md]: Ok then I move that up to Added

**Pull request merged**